styles/Footer.module.css

```diff
@@ -96,3 +96,54 @@
   color: var(--text-default);
   text-decoration: none;
 }
+@media (max-width: 1023px) {
+  .footer_top__container {
+    flex-direction: column-reverse;
+  }
+  .footer_top_taotech__container {
+    padding-top: 48px;
+    height: 100%;
+    width: 100%;
+    border-style: none;
+    align-items: center;
+    text-align: center;
+  }
+  .footer_top_links__container {
+    margin-left: 0;
+  }
+  .footer_top_link__container {
+    margin-right: 0;
+    flex: 1;
+  }
+  .footer_address {
+    padding-bottom: 60px;
+  }
+}
+@media screen and (max-width: 743px) {
+  .footer_top_taotech__container {
+    padding-top: 95px;
+  }
+  .footer_top_links__container {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 48px;
+  }
+  .footer_top_link__ul {
+    align-items: center;
+  }
+  .footer_bottom__container {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+  .footer_bottom__privacy {
+    position: static;
```

Comment:
Looks good, but may not necessarily position these elements using "absolute" before and "static" here. Would putting them in a flexbox and use "space-between" an easier way?

Reply:
Thanks